src/Microsoft/Windows/ComputeVirtualization/HcsFunctions.cs

```diff
@@ -35,6 +35,6 @@ internal class HcsFunctions
         public static extern int TerminateProcessInComputeSystem(string id, int pid);
 
         [DllImport("vmcompute.dll", PreserveSig = false, ExactSpelling = true)]
-        public static extern void HNSCall(string method, string path, string request, [MarshalAs(UnmanagedType.LPWStr)] out string response);
+        public static extern int HNSCall(string method, string path, string request, [MarshalAs(UnmanagedType.LPWStr)] out string response);
```

Comment:
I don't think you need to do this. As-is, it will throw an exception if `HNSCall` returns < 0. That's what `PreserveSig = false` does.